src/ensembl/src/shared/components/error-screen/GeneralErrorScreen.tsx

```diff
@@ -16,7 +16,10 @@
 
 import React from 'react';
 
-import { HomeLink, ReleaseVersion, Copyright } from 'src/header/Header';
+import { ReleaseVersion, Copyright } from 'src/header/Header';
+// import { HomeLink } from 'src/header/header-buttons/HeaderButtons';
```

Comment:
Can this be removed? also on live 31

Reply:
I've removed it. In fact, I've now removed the whole HeaderButtons component.